tests/src/Kernel/ViewsLinkFieldsTest.php

```diff
@@ -58,17 +58,16 @@ protected function setUp(): void {
         // Install user module to avoid user 1 permissions bypass.
         \Drupal::moduleHandler()->loadInclude('user', 'install');
         user_install();
-
-        // Create two medias to check access with different scopes, 'any' and 'own'.
-        $this->createMediaEntity('document');
-        $this->ownMedia = $this->createMediaEntity('document');
-        ;
     }
 
     /**
      * Tests link fields.
      */
     public function testLinks(): void {
+        // Create two medias to check access with different scopes, 'any' and 'own'.
+        $this->createMediaEntity('document');
+        $this->ownMedia = $this->createMediaEntity('document');
+
```

Comment:
We could rethink the information flow in the test, but it could happen that by doing this we introduce more complexity in other ways.
So now I leave it to you, you can keep the last commit or drop it, whatever feels better to you.

Reply:
We keep it as is for the moment.